### DISCUSSION TITLE
Configuration for a private mirror

### DISCUSSION THREAD

**Discussion by jcralbino** [sdkman/sdkman-cli-native#197] [Q&A]
Hello
If I understand correctly the clients running this tool  are required to have access to the internet ( proxy is supported), and the downloads will be done from the original sites ( the files are not hosted in a single domain)

If this is correct, I would like to know if we can use a private mirror, or if there is a configuration that allows the use of a single domain to download the files 

Regards 
Jose

**Comment by marc0der** [comment]
That's right, we are merely a broker to where the actual download archives live. We don't host anything ourselves.

If you wanted to spin up your own services and datastore pointing to a mirror it could be done, although this would take a lot of effort to achieve.

**Reply by jcralbino** [reply]
Is there an easy way to identify the urls or domains needed to allow in the proxy for specific installations?
